src/Components/common/LoadingState/Loading/index.js

```diff
@@ -0,0 +1,10 @@
+import { IconSpinner, LoadingText, IconContainer } from "./styled";
+
+export const Loading = () => (
```

Comment:
Tu zmienił bym nazwę, `loading` to stan a komponent nazwał bym po prostu `Loader`

Reply:
Sorki nie moge odpowiedziec na twoj glowny komentarz wiec zostawie odpowiedz tutaj: ten folder Components byl stworzony na samym poczatku razem z aplikacja jesli dobrze pamietam wiec chyba to wy dodaliscie ten drugi common :)

Reply:
Popieram zmianę nazwy na Loader ;)

Reply:
Przeglądając pierwsze commity to widzę, że nie było ani folderu Componets ani common. Przechodząc dalej to Magalena i ja wrzuciliśmy nasze prace po prostu w common, a przy twoim 3 commicie został stworzony doatkowy folder Components, musiało nam wszystkim to umknąć ;D Osobiście proponuję Ci przenieść wszystko po prostu do common, zacomitować tą zmianę i wydaję mi się że będzie git ;D